invenio_github/templates/semantic-ui/invenio_github/settings/index.html

```diff
@@ -69,7 +69,7 @@ <h2>
             <div class="column"></div>
             <div class="column centered">
               <br>
-              <a class="ui label blue" href="https://doi.org/10.5281/zenodo.8475">
+              <a href="https://doi.org/10.5281/zenodo.8475">
```

Comment:
minor/shelve: it seems like we're hardcoding Zenodo stuff in this repo... it's a bit unavoidable at this point since there's no alternative option for demonstrating what the GitHub integration looks like. Let's shelve this so that we remove Zenodo references in the future.

Reply:
Thanks for noticing! I will add the `TODO` here for this